src/transformer/mockDefiner/cache/factoryDeclarationListCache.ts

```diff
@@ -0,0 +1,44 @@
+import * as ts from 'typescript';
+
+export interface DeclarationCache {
```

Comment:
we already have a DeclarationCache class in declarationCache.ts, can we call this or that differently?

Reply:
I've renamed the class to DeclarationListCache to follow the same name conventions and I've renamed the interface to DeclarationListCacheElement.